build.yaml

```diff
@@ -10,12 +10,24 @@ cassandra:
 build:
   - type: maven
     version: 3.2.5
-    goals: clean compile test --fail-never -Plong
+    goals: clean compile test integration-test --fail-never -Plong
```

Comment:
I think it is recommended to invoke the verify phase instead, like in `clean verify --fail-never -Plong`